src/e2e/sdk_remote_integration.spec.ts

```diff
@@ -1,204 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Maybe this PR is a good opportunity to rename the e2e test file to something simpler like `test.spec.ts` or `sdk_end2end.spec.ts`? I don't get the `remote` in its file name.

Reply:
Now called `sdk_e2e.spec.ts`.